blog/ga-for-neon.adoc

```diff
@@ -0,0 +1,101 @@
+= JBoss Tools and Red Hat Developer Studio for Eclipse Neon
+:page-layout: blog
+:page-author: akazakov
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+:page-date: 2016-06-27
+
+link:/downloads/jbosstools/neon/4.4.0.Final.html[JBoss Tools 4.4] and link:/downloads/devstudio/neon/10.0.0.GA.html[Red Hat JBoss Developer Studio 10.0] for Eclipse Neon are here waiting for you. Check it out!
+
+image::/blog/images/devstudio10.png[]
+
+== Installation
+
+JBoss Developer Studio comes with everything pre-bundled in its installer. Simply download it from our https://www.jboss.org/products/devstudio.html[JBoss Products page] and run it like this:
+ 
+    java -jar jboss-devstudio-<installername>.jar
+
+JBoss Tools or Bring-Your-Own-Eclipse (BYOE) JBoss Developer Studio require a bit more:
+
+This release requires at least Eclipse 4.6 (Neon) but we recommend
+using the latest http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/neon[Eclipse 4.6 Neon JEE Bundle] since then you get most of the dependencies preinstalled. 
```

Comment:
Current URL is http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/neonr and not http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/neon
Maybe this will change when released but current link generates 404